Product.js

```diff
@@ -0,0 +1,41 @@
+export class Product {
+  constructor({name,description,price,tags,images}){
+    this.name = name;
+    this.description = description;
+    this.price = price;
+    this._tags = Array.isArray(tags) ? [...tags] : [];
+    this._images = Array.isArray(images) ? [...images] : [];
+    this.favoriteCount=0;
+    }
+  favorite() {
+    this.favoriteCount+=1
+    }
+  
+  get price () {
+    return `가격은 ${this._price}원 입니다`;
+  }

```

Comment:
실제 가격을 알려주는 String이 리턴되고있는데, 만약 product.price * 2같은걸 하면 어떻게될까요?
getter에서는 원본 필드가 리턴되는게 가장 바람직합니다. 정말 필요하다면, 다른 함수를 만들어서 사용하는 게 좋아요.

```suggestion
  get price () {
    return this._price;
  }
  
  getPriceText() {
    return `가격은 ${this._price}원 입니다`;
  }

```